src/components/SQFormTransferTool/AccordionBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  Section,
+  SectionBody,
+  SectionHeader,
+  TextButton,
+} from 'scplus-shared-components';
+import type {OnTransfer, TransferProduct} from './types';
+
+export type AccordionBodyProps = {
+  transferProduct: TransferProduct;
+  onTransfer: OnTransfer;
+};
+
+export default function AccordionBody({
+  transferProduct,
+  onTransfer,
+}: AccordionBodyProps): React.ReactElement {
+  const {productDisplayName, modalLinkText} = transferProduct;
```

Comment:
I would vote to keep this as link text instead of changing to transfer text like you mentioned in your video.